### ISSUE TITLE
i always get wrong pictures use pretrained model. the output is as below:

### ISSUE THREAD

**Issue by sporterman** [lyndonzheng/Pluralistic-Inpainting#6]

**Comment by sporterman** [comment]
![image](https://user-images.githubusercontent.com/13059352/55381395-a5938380-5555-11e9-975d-63cf4a3dfe61.png)

**Comment by sporterman** [comment]
what's the problem?

**Comment by lyndonzheng** [comment]
@sporterman I guess you use a wrong pre-tained model or a wrong mask input. Can you explain the testing details you used? You use the GUI ui_main.py or test.py.

**Comment by sporterman** [comment]
@lyndonzheng It is GUI result,   I've found that the mask file through GUI is gray picture, not binary picture. Is there any prblem,  i'm sure the model were correct, because i use the final place2-random-form model, and random input the picture,  can you give me some advice?

**Comment by sporterman** [comment]
@lyndonzheng even thought i use center models, the result were the same. i doubt is there any parameters i should care?

**Comment by lyndonzheng** [comment]
@sporterman This test image comes from the Celeba-HQ dataset, you should use the celeba-HQ-random pretrained model for testing.

**Comment by sporterman** [comment]
@lyndonzheng 
![image](https://user-images.githubusercontent.com/13059352/55866961-0c4b1980-5bb4-11e9-8687-6004c3e53138.png)

**Comment by sporterman** [comment]
the random buttom just random choose pictures from correspond path

**Comment by lyndonzheng** [comment]
@sporterman I think you still did not correctly load the pre-trained model. I just download the released code and pre-trained model, the results are correct. Maybe you can try to download the pre-trained model again.

**Comment by sporterman** [comment]
@lyndonzheng again....
![image](https://user-images.githubusercontent.com/13059352/55869428-0f94d400-5bb9-11e9-85c7-5a5520826dce.png)

**Comment by sporterman** [comment]
@lyndonzheng what's you experiment environment?   i'm using  pytorch 1.0.1.post2, ubuntu 16.04, python3.5

**Comment by lyndonzheng** [comment]
@sporterman The original code was trained on pytorch 0.4, ubuntu16.04 and python3.6. I also test the model on pytorch1.0, ubuntu16.04 and python3.6. I think you results are totally random resutls. You model weight maybe is randomly initialized instead of the pretrained-model weight.

**Comment by sporterman** [comment]
@lyndonzheng when using this command: python3 test.py --name celeba_random --img_file ./datasets/celeba-hq/, result:
![image](https://user-images.githubusercontent.com/13059352/55928413-1e2bcb80-5c4b-11e9-9aa5-c43bae93e6ce.png)

![image](https://user-images.githubusercontent.com/13059352/55928371-fc324900-5c4a-11e9-88fd-b274552d1e22.png)
were these model loaded corrected?

**Comment by sporterman** [comment]
@lyndonzheng It was solved, This code maybe have some bugs while running on CPU .  All things went well on CPU machine. much thanks for your quickly answers.

**Comment by lyndonzheng** [comment]
@sporterman Yes, the pre-trained model can only be tested on GPU. That's the problem of model_saving function. We will modify this part in the next version.

**Comment by sunjunlishi** [comment]
when cpu  version
